scripts/validate_metadata.py

```diff
@@ -0,0 +1,238 @@
+import sys
+from itertools import pairwise
```

Comment:
note that `pairwise` does not exist in the the `itertools` provided in python `3.9`, this should be resolved once you bump the workflow to 3.11